p2p/transport/webrtc/pb/Makefile

```diff
@@ -0,0 +1,9 @@
+pbgos := $(patsubst %.proto,%.pb.go,$(wildcard *.proto))
```

Comment:
Given tis is a go project, why not simply use `go:generate` functionality instead of Makefiles? That's at least what I usually did for Go projects, keeping Makefile for my C/C++ projects (if not using something else there).

Reply:
I fully agree, on we're in the process of switching to `go generate` for our Protobufs: https://github.com/libp2p/go-libp2p/pull/1979.

I expect that PR to land before this one, I suggest we rebase this PR once #1979 is merged, and get rid of the Makefile then.